### PR TITLE
Bug 1195335 - Split wiki views into separate modules.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

**Comment by jwhitlock** [comment]
Code looks good.  Moving on to testing the views.

File: kuma/wiki/views/misc.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+            flag, created = ContentFlag.objects.flag(
+                request=request, object=doc,
+                flag_type=flag_type,
+                explanation=form.cleaned_data['explanation'],
+                recipients=recipients)
+            return redirect(doc)
+    else:
+        form = DocumentContentFlagForm(data=request.GET)
+
+    return render(request, 'wiki/flag.html', {'form': form, 'doc': doc})
```
Comment (jwhitlock) [inline kuma/wiki/views/misc.py]: This file doesn't exist.  I don't think this PR removed or misplaced it.

Comment (jezdez) [inline kuma/wiki/views/misc.py]: We *just* had an error in stage about this, odd.

Comment (jezdez) [inline kuma/wiki/views/misc.py]: Ah, https://github.com/mozilla/kuma/commit/b9f1a32cff26a1b3ecee7fbe6c0bcb5ac6afb394 added the view, but no template? @groovecoder do you remember any of this?

**Comment by jwhitlock** [comment]
Practical tests pass, with some issues that appear to exist before this change:

* [bug 1197384](https://bugzilla.mozilla.org/show_bug.cgi?id=1197384) - Unable to view /admin/wiki/revision/ in vagrant environment
* [bug 1197386](https://bugzilla.mozilla.org/show_bug.cgi?id=1197386) - No template for $flag page action
* [bug 1197390](https://bugzilla.mozilla.org/show_bug.cgi?id=1197390) - 500 ISE for GET $purge

**Pull request merged**